src/types.ts

```diff
@@ -34,9 +34,20 @@ export interface TsJestGlobalOptions {
    *     - `false`: do NOT use default config file
    *     - `path/to/tsconfig.json`: path to a specific tsconfig file (<rootDir> can be used)
    *     - `{...}`: an object with inline compiler options
+   * @alias tsconfig
    */
   tsConfig?: boolean | string | _ts.CompilerOptions
 
+  /**
+   * Compiler options. It can be:
+   *     - `true` (or `undefined`, it's the default): use default tsconfig file
+   *     - `false`: do NOT use default config file
+   *     - `path/to/tsconfig.json`: path to a specific tsconfig file (<rootDir> can be used)
+   *     - `{...}`: an object with inline compiler options
+   * @alias tsConfig
+   */
+  tsconfig?: boolean | string | _ts.CompilerOptions
```

Comment:
@ahnpnl do you think it makes sense to actually add this to the options and document it or should it be more of a silent feature (not the golden path)? I just think from an Intellisense perspective, it could be a bit confusing to see both `tsConfig` and `tsconfig` pop up in your face.

Reply:
Personally I think if a feature is available to users, it shouldn’t be hidden from users. You can adjust the doc saying like for example, the table of configuration property, tsConfig (or alias tsconfig). Shouldn’t be too much of modifying the docs but it’s clear to users :)